src/views/AdvancedSearchView/components/PropertyFilter/index.js

```diff
@@ -159,8 +159,7 @@ const PropertyFilter = ({
           } else {
             newPropertyModel = null;
           }
-        } else if (newPropertyModel.type === 'link') {
-          newPropertyModel.type = 'linkset';
+        } else if (newPropertyModel.type === 'linkset') {
```

Comment:
If this is not set, you can no longer filter by 2+ relevances. (I've deployed this to development for testing). We'll need to fix the operator rather than changing the type here. Should be ok with a list if the operator is 'IN'

Reply:
Reverted my original change and I've pushed what you've described